app/models/concerns/domain/force_delete.rb

```diff
@@ -152,4 +153,16 @@ def allow_deletion
   def force_delete_fast_track_start_date
     Time.zone.today + Setting.expire_warning_period.days + Setting.redemption_grace_period.days
   end
+
+  def notify_parties(reason)
+    ent = notification_template if %w[ENTITY_BURIED EMAIL].include? reason
+    ent ||= reason.downcase
+    self.template_name = ent
```

Comment:
Are you sure about that? Not `notification_template`, but `reason.downcase`?

Reply:
This was supposed to be for any template that's not private_person or legal_person, which had to be defined in given csv. As I noticed `notification_template` tries to decide on template on it's own, I think it can be dropped for now. Hope it's going to work as supposed to before we're going to scan all contacts post-create for invalid mails / phones.